app/views/indices/_form.html.haml

```diff
@@ -1,8 +1,9 @@
 - first_category = nil
 = simple_form_for([@community,@index], html: { class: 'vulnerability-index-form'}) do |f|
-  .panel.panel-default
+  .panel.panel-default{ style: 'margin-top: 20px;' }
     .panel-heading
-      %h1.panel-title= "#{@index.community.name} - #{@index.awrvi_version.name}"
+      %h2{ style: 'margin: 0px;' }
+        AWRVI Form for #{link_to @index.community.name, @index.community}
```

Comment:
@vlraymond You (I think it was you) had a better term than 'Form', since that usually doesn't have meaning to the target audience.